src/modules/languages/typst.nix

```diff
@@ -30,6 +35,6 @@ in
       pkgs.typstyle # formatter
     ];
 
-    env.TYPST_FONT_PATHS = lib.concatStringsSep ":" cfg.fontPaths;
+    env.TYPST_FONT_PATHS = if cfg.fontPaths then (lib.concatStringsSep ":" cfg.fontPaths) else null;
```

Comment:
I think this needs to check for empty list

Reply:
You are right, forgot that nix does not autocast to bool.